### PR TITLE
.NET 9: OpenIdConnectHandler adds support for Pushed Authorization Requests (PAR) 

### PR DESCRIPTION
**Pull request by damienbod**
fixes https://github.com/dotnet/AspNetCore.Docs/issues/33332

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/authentication/claims.md](https://github.com/dotnet/AspNetCore.Docs/blob/108625b884f5d29b431226954cbec7069a996d4d/aspnetcore/security/authentication/claims.md) | [Mapping, customizing, and transforming claims in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/security/authentication/claims?branch=pr-en-us-33366) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@damienbod  looks great.
@josephdecock please review.

File: aspnetcore/security/authentication/claims.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ Another way to get the user claims is to use the OpenID Connect User Info API. T
...
+
+:::code language="csharp" source="~/release-notes/aspnetcore-9/samples/PAR/Program.cs" id="snippet_1" highlight="8-99":::
+
+To ensure that authentication only succeeds if PAR is used, use [PushedAuthorizationBehavior.Require](https://source.dot.net/#Microsoft.AspNetCore.Authentication.OpenIdConnect/PushedAuthorizationBehavior.cs,3af73de8f33b70c5) instead. This change also introduces a new [OnPushAuthorization](https://source.dot.net/#Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/OpenIdConnectEvents.cs,6a21c8f3a90753c1) event to [OpenIdConnectEvents](/dotnet/api/microsoft.aspnetcore.authentication.openidconnect.openidconnectevents) which can be used customize the pushed authorization request or handle it manually. See the [API proposal](https://github.com/dotnet/aspnetcore/issues/51686) for more details.
```
Comment (josephdecock) [inline aspnetcore/security/authentication/claims.md:46]:
```suggestion
To ensure that authentication only succeeds if PAR is used, use [PushedAuthorizationBehavior.Require](https://source.dot.net/#Microsoft.AspNetCore.Authentication.OpenIdConnect/PushedAuthorizationBehavior.cs,3af73de8f33b70c5) instead. This change also introduces a new [OnPushAuthorization](https://source.dot.net/#Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/OpenIdConnectEvents.cs,6a21c8f3a90753c1) event to [OpenIdConnectEvents](/dotnet/api/microsoft.aspnetcore.authentication.openidconnect.openidconnectevents) which can be used to customize the pushed authorization request or handle it manually. See the [API proposal](https://github.com/dotnet/aspnetcore/issues/51686) for more details.
```

**Review by josephdecock — CHANGES_REQUESTED** [review]
Otherwise looks good!

File: aspnetcore/security/authentication/claims.md (hunk 1 above)
Comment (damienbod) [reply aspnetcore/security/authentication/claims.md]: fixed, thanks for the feedback

**Pull request merged**